modules/apps/frontend-editor/frontend-editor-alloyeditor-web/src/main/java/com/liferay/frontend/editor/alloyeditor/web/internal/editor/configuration/BaseAlloyEditorConfigContributor.java

```diff
@@ -63,17 +63,16 @@ public void populateConfigJSONObject(
 		).put(
 			"extraPlugins",
 			"addimages,ae_dragresize,ae_imagealignment,ae_placeholder," +
-				"ae_selectionregion,ae_tableresize,ae_tabletools,ae_uicore," +
-					"autolink"
+				"ae_selectionregion,ae_tableresize,ae_tabletools,ae_uicore"
 		).put(
 			"imageScaleResize", "scale"
 		).put(
 			"language",
 			StringUtil.replace(getLanguageId(themeDisplay), "iw_", "he_")
 		).put(
 			"removePlugins",
-			"contextmenu,elementspath,floatingspace,image2,link,liststyle," +
-				"resize,table,tabletools,toolbar"
+			"autolink,contextmenu,elementspath,floatingspace,image2,link," +
```

Comment:
Downloading and running this fix it looks fine to get back the "native" alloy-editor linking panel but I have one thing to be aware: this ck-editor autolink plugin was added there for preventing a performance issue [as this commit explain](https://github.com/liferay-frontend/liferay-portal/commit/11dfcd178398dc0798b756a4f4a141352397bec8)

[This ticket](https://issues.liferay.com/browse/LPS-119875) explain more about this performance issue but testing locally with this current fix it looks already solved. See the following video:


https://user-images.githubusercontent.com/7663513/120027417-b91dc600-bfc9-11eb-9301-e2fac70e2d0a.mov



Hey @john-co or @nkkwon, could someone just see from a QA point of view here?

Reply:
Hi @diegonvs , I took a look and the performance issue looks resolve in this PR. Looks ok from QA standpoint.

https://user-images.githubusercontent.com/35235266/120045620-47437d80-bfc5-11eb-9ff2-a014af46deba.mp4